src/index.js

```diff
@@ -181,6 +194,27 @@ const App = () => {
         label="Checkbox"
       />
 
+      <ColorPalette
+        colors={DEFAULT_COLORS}
+        selectedColor={DEFAULT_COLORS[0]}
+        handleColorChange={() => { }}
+      />
+
+      <ColorPicker
+        handleChange={value => { }}
+        color="#fefefe"
+        active={true}
+      />
+
+      <Accordion className="mt-2" defaultActiveKey={0}>
+        <Accordion.Item title="Opened Heading" isOpen>
```

Comment:
No need to pass `isOpen`. It is handled internally.

Reply:
This is an example an item can be opened by default.